lib/vagrant-libvirt/cap/synced_folder_virtiofs.rb

```diff
@@ -0,0 +1,110 @@
+require 'log4r'
+require 'ostruct'
+require 'nokogiri'
+require 'digest/md5'
+
+require 'vagrant/util/subprocess'
+require 'vagrant/errors'
+require 'vagrant-libvirt/errors'
+# require_relative "helper"
+
+module VagrantPlugins
+  module SyncedFolderVirtioFS
+    class SyncedFolder < Vagrant.plugin('2', :synced_folder)
+      include Vagrant::Util
+      include VagrantPlugins::ProviderLibvirt::Util::ErbTemplate
+
+      def initialize(*args)
+        super
+        @logger = Log4r::Logger.new('vagrant_libvirt::synced_folders::virtiofs')
+      end
+
+      def usable?(machine, _raise_error = false)
+        # bail now if not using Libvirt since checking version would throw error
+        return false unless machine.provider_name == :libvirt
+
+        # <filesystem/> support in device attach/detach introduced in 1.2.2
+        # version number format is major * 1,000,000 + minor * 1,000 + release
+        libvirt_version = machine.provider.driver.connection.client.libversion
+        libvirt_version >= 6_006_000
```

Comment:
In the readme updates it's suggested support was added in 6.2.0 of libvirt, should this check use that version or is there some issues with the earlier releases that it's better to skip this until 6.6.0? Perhaps in that case, worth adjusting the README notes.

Reply:
Updated as 6.2.0 (Ooops since I am using Ubuntu 21.04 so I keep thinking of 6.6.0....)